README.rst

```diff
@@ -1,72 +1,83 @@
-==================================
-The Python Component System
+=======================================
+The Python Component System and AgentOS
+=======================================
+
+This project consists of two major pieces: the Python Component System (PCS)
+and AgentOS.
+
+Python Component System (PCS)
 ==================================
 
-The Python Component System (PCS) is an open source **Python API, command line
-interface, and public web registry** for building, running, and sharing Python
-programs.  The Python Component System is designed to:
-  * Make program execution reproducible.
+PCS is an open source **Python API, command line interface, and public web
+registry** for building, running, and sharing Python programs.  PCS:
```

Comment:
Do these bullets overstate our current status/progress a little? :) Maybe we can call these the goals of PCS?

Reply:
We want to over-promise and under-deliver, right?  Or was it the other way... (jk will fix)